libs/shared/src/lib/components/ui/aggregation-builder/aggregation-builder.component.html

```diff
@@ -25,7 +25,7 @@
     <!-- Pipeline builder -->
     <div class="flex grow gap-2">
       <shared-pipeline
-        class="basis-2/3 grow"
+        class="basis-2/3 grow overflow-hidden"
```

Comment:
@RenzoPrats 

took me some time to find some time to review it
this is the fix for the incorrect sizing of editors when displaying both custom aggregation & preview

no need to have listeners on resize etc, this is fixing everything